### DISCUSSION TITLE
Set Mtime and Atime of symbolic link

### DISCUSSION THREAD

**Discussion by araneolus** [hierynomus/sshj#807] [Q&A]
Hi,

Is it possible to create a remote symbolic link and set the atime and mtime to anohter time as the create time.
I have tried it with the SFTPEngine setAttributes method (SESTATUS) and the RemoteFile set Attributes method (FSESTATUS) but it doesn't work. 
Perhapse it is not possible by the ssh Daemon ??

**Comment by hierynomus** [comment]
What part doesn't work? Do you get an exception? They're just ignored?

**Comment by araneolus** [comment]
There is no error. I am able to create the symbolic link with
`this.sftpClient.symlink(target, newLink);`
and than set the file Attributes to the link with
`this.sftpClient.getSFTPEngine().setAttributes(file,attrs);`
The Values of FileAttributes  are
`[uid=0,gid=1001;atime=1663518244,mtime=1663518244]`
and the file path is
`/etc/letsencrypt/live/hupssotest.alfa-cloud.net/cert.pem`
After this, the link has still the root uid, and root gid (uid=0,gid=0)
`lrwxrwxrwx  1 root root          49 Sep 18 18:58 cert.pem -> ../../archive/hupssotest.alfa-cloud.net/cert8.pem`
There are no exception during execution.
It is the same, if I execute a chmod to the link without the '-h' option at linux.

I have also tried to override the setAttributes funktion with FSETSTAT, without success :-(
```
    public void setAttributes(String path, FileAttributes attrs)
            throws IOException {
        doRequest(
                newRequest(PacketType.FSETSTAT).putString(path, sub.getRemoteCharset()).putFileAttributes(attrs)
        ).ensureStatusPacketIsOK();
    }

```
